### PR TITLE
Weekly PR

### PR DESCRIPTION
**Pull request by EugeneButusov**

### CODE REVIEW

**Comment by EugeneButusov** [comment]
@borisspektor @refaelos, it's ready to merge.

File: README.md [hunk 1]
```diff
@@ -23,6 +23,10 @@ SOOMLA modules internally use these features, though we encourage you to use the
 
 [soomla-unity3d-core 1.1.0](http://library.soom.la/fetch/unity3d-core/1.1.0?cf=github)
 
+#### From sources:
```
Comment (borisspektor) [inline README.md]: @EugeneButusov please remove the `;` at the end of lines 1 and 2

File: Soomla/Assets/Plugins/Soomla/Core/Config/CoreSettings.cs [hunk 2]
```diff
@@ -36,6 +36,9 @@ public class CoreSettings : ISoomlaSettings
 		public static string DB_KEY_PREFIX = "soomla.";
 
 		static CoreSettings instance = new CoreSettings();
+
+		static string currentModuleVersion = "1.0.10";
```
Comment (borisspektor) [inline Soomla/Assets/Plugins/Soomla/Core/Config/CoreSettings.cs]: @EugeneButusov latest version is 1.1.0

**Comment by EugeneButusov** [comment]
@borisspektor, fixed it.

**Pull request merged**